ciaox11/core/ciaox11_container.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file    ciaox11_container.cpp
+ * @author  Martin Corino
+ *
+ * @brief   Base class for CIAOX11 container implementation(s).
+ *
+ * @copyright Copyright (c) Remedy IT Expertise BV
+ */
+
+#include "ciaox11/core/ciaox11_container.h"
+
+namespace CIAOX11
+{
+    Container::Container (std::string name)
+     : name_ (name)
```

Comment:
``` suggestion
     : name_ (std::move(name))
```

Reply:
compiler should do this implicitly since this is a value parameter